.github/workflows/publish.yml

```diff
@@ -1,4 +1,5 @@
-name: Build and push generic producer to docker hub
+# Build and push generic producer to docker hub
+name: publish.yaml
```

Comment:
Do you still want the name? I know you prefer when you can see the file path instead.
```suggestion
```

Reply:
Yeah, I like having just the filename